orga/app/components/tube/tube.hbs

```diff
@@ -1,12 +1,21 @@
 <td class="table__column--center">
-  <Input id="tube-{{@tube.id}}" @type="checkbox" @checked={{this.checked}} />
+  <PixCheckbox
+    @screenReaderOnly={{true}}
+    @id="tube-{{@tube.id}}"
+    {{on "click" this.toggleTube}}
+    @checked={{this.checked}}
+  >
+    <:label>
+      {{@tube.practicalTitle}}
+      :
+      {{@tube.practicalDescription}}
+    </:label>
+  </PixCheckbox>
 </td>
 <td>
-  <label for="tube-{{@tube.id}}">
-    {{@tube.practicalTitle}}
-    :
-    {{@tube.practicalDescription}}
-  </label>
+  {{@tube.practicalTitle}}
+  :
+  {{@tube.practicalDescription}}
```

Comment:
**To do :** retirer ces 3 lignes ci-dessus

Reply:
edit : il faut les garder . ils sont dans un TD à part. et le label de la checkbox est en screenReaderOnly ( non visible à l'écran ) . c'est pas l'idéal en terme d'accessibilité. mais je reste ISO avec ce qui était fait.